### PR TITLE
Reapply corrections to various models

### PR DESCRIPTION
**Pull request by ssuchacz**

### CODE REVIEW

File: sbml/iMA871/exchange.yaml [hunk 1]
```diff
@@ -312,6 +312,9 @@ compounds:
   upper: 0.0
 - id: M_PROPALe
   reaction: '1380'
+- id: M_BIOMASS
+  compartment: C-c
```
Comment (keithdt) [inline sbml/iMA871/exchange.yaml]: Can we double check the formatting of this and make sure that the compartment designation is correct. Its usually C_c not C-c

Comment (keithdt) [reply sbml/iMA871/exchange.yaml]: Also now the compartment should just be 'c' not with the C_c due to the improvements to PSAMM-import parsing.

**Review by keithdt — COMMENTED** [review]
The only needed change i saw was the compound designation for biomass in the iMA871 model.

File: sbml/RECON1/reactions.yaml [hunk 2]
```diff
@@ -10018,7 +10018,7 @@
   equation: atp[n] + ump[n] <=> adp[n] + udp[n]
 - id: UNK2
   name: UNK2
-  equation: 2kmb[c] + gln__L[c] + h[c] => glu__L[c] + met__L[c]
+  equation: M_2kmb[c] + M_gln__L[c] + (2) M_h_c[c] => M_glu__L[c] + M_met__L[c]
```
Comment (jonls) [inline sbml/RECON1/reactions.yaml]: @ssuchacz I think the `M_` prefix, and on `M_h_c` the `_c` suffix, should not be added. These prefixes/suffixes are no longer part of the model so your change should not reintroduce them. The change here should just be to add the additional `h[c]`.

**Review by jonls — CHANGES_REQUESTED** [review]

**Comment by jonls** [comment]
@ssuchacz Keith noticed that you had applied the changes to the iSyn731 from the commit I specified. Those changes were not actually manual corrections but were just a result of importing the model from a newer file. It looks like the changes you made were correct but a better/safer way to do this was to simply reimport the correct model file. I'm sorry I didn't notice this earlier. I have reimported iSyn731 from the correct file now so that is why it says that your branch has conflicts. You can ignore this message for now.

**Review by jonls — COMMENTED** [review]
This looks good now. It would be nice if we could split this into separate commits, one for each model. This would then allow us to refer to specific commits for each model in the README file. You can ask me later if you don't know how to do this.

**Pull request closed without merging**